overdrive.py

```diff
@@ -890,12 +890,8 @@ def process_item(self, identifier):
             e = "Could not extract metadata from Overdrive data: %r" % info
             return CoverageFailure(self, identifier, e, transient=True)
 
-        result = self.set_metadata(
+        return self.set_metadata(
             identifier, metadata, 
             metadata_replacement_policy=self.metadata_replacement_policy
         )
 
-        if not isinstance(result, CoverageFailure):
-            # Success!
-            result = self.set_presentation_ready(result)
-        return result
```

Comment:
This is no longer necessary since the superclass process_batch() takes care of setting the work as presentation-ready.